arcgis-ios-sdk-samples/AppDelegate.swift

```diff
@@ -26,33 +26,15 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UISplitViewControllerDele
     var categoryBrowserViewController: ContentCollectionViewController {
         return (splitViewController.viewControllers.first as! UINavigationController).viewControllers.first as! ContentCollectionViewController
     }
-    
-    /// The URL of the content plist file inside the bundle.
-    var contentPlistURL: URL {
-        return Bundle.main.url(forResource: "ContentPList", withExtension: "plist")!
-    }
-    
-    /// Decodes an array of categories from the plist at the given URL.
-    ///
-    /// - Parameter url: The url of a plist that defines categories.
-    /// - Returns: An array of categories.
-    func decodeCategories(at url: URL) -> [Category] {
-        do {
-            let data = try Data(contentsOf: url)
-            return try PropertyListDecoder().decode([Category].self, from: data)
-        } catch {
-            fatalError("Error decoding categories at \(url): \(error)")
-        }
-    }
-
+ 
     func application(_ app: UIApplication, open url: URL, options: [UIApplication.OpenURLOptionsKey : Any] = [:]) -> Bool {
         if url.absoluteString.range(of: "auth", options: [], range: nil, locale: nil) != nil {
             AGSApplicationDelegate.shared().application(app, open: url, options: options)
         }
         return true
     }
 
-    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey : Any]? = nil) -> Bool {
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
```

Comment:
Though it doesn't seem to make a whole log of sense, the default value for launchOptions is part of the API and is part of the autocompleted function signature. Please let it the way it was.

Reply:
This was accidental when merging in v.next.